classes/ActionScheduler_NullAction.php

```diff
@@ -3,14 +3,14 @@
 /**
  * Class ActionScheduler_NullAction
  */
-class ActionScheduler_NullAction extends ActionScheduler_Action {
+class ActionScheduler_NullAction extends ActionScheduler_AbstractAction {
```

Comment:
If possible, this should be `ActionScheduler_Null_Action` for consistency.

Reply:
`ActionScheduler_NullAction` has been in AS since v1.0. It _is_ the base for what others should be consistent with. 🙂